core/ipc/rpc/camel/src/main/java/org/opennms/core/rpc/camel/CamelRpcClientFactory.java

```diff
@@ -184,9 +205,14 @@ public void setLocation(String location) {
     public void start() {
         tracerRegistry.init(SystemInfoUtils.getInstanceId());
         tracer = tracerRegistry.getTracer();
+        // Initialize metrics reporter.
+        metricsRepoter = JmxReporter.forRegistry(metrics).
+                inDomain(JMX_DOMAIN_RPC).build();
+        metricsRepoter.start();
     }
 
     public void stop() {
         //pass
+        metricsRepoter.close();
```

Comment:
Can we make sure metricsRepoter is not null before closing? Calling stop before calling start would throw an NPE.